index.html

```diff
@@ -67,30 +74,55 @@ <h2>
     <h2>
       Model
     </h2>
-    <dl class="definitions" data-sort="" data-cite="vc-data-model">
+
+    <p class="issue" title="Definitions under discussion">
+The goal of the definitions in this section is to re-use or establish
+terminology that is common across a variety of digital credential formats and
+protocols.
+    </p>
+
+    <dl class="definitions" data-sort="" data-cite="vc-data-model-2.0">
       <dt>
         <dfn>Digital credential</dfn>
       </dt>
       <dd>
-        <p>
-          Is a [=verifiable credential=] about a person.
-        </p>
+        A cryptographically secured digital document containing one or more
```

Comment:
I'm uncomfortable with the with the word "secured" here... I'm not sure cryptographically necessarily means something is secure. 
```suggestion
        A cryptographically authenticated digital document containing one or more
```

Reply:
https://datatracker.ietf.org/doc/html/rfc4949 has some terminology you might consider:

```
credential
      1. (I) /authentication/ "identifier credential": A data object
      that is a portable representation of the association between an
      identifier and a unit of authentication information, and that can
      be presented for use in verifying an identity claimed by an entity
      that attempts to access a system. Example: X.509 public-key
      certificate. (See: anonymous credential.)
```

Reply:
```suggestion
        A cryptographically signed digital document containing one or more
```

Reply:
Oh! "singed" is spot on, @Sakurann! Let's go with that.